apps/lfe-ls/src/lsp-proc.lfe

```diff
@@ -212,6 +212,12 @@ Handler functions (like `%on-initialize-req`) are expected to return:
            ,state
            null)))))
 
+(defun extract-position-data
```

Comment:
Excellent.
Can you please prefix a "%" character to indicate internal function.

Reply:
Done. :)